### ISSUE TITLE
Link without an href takes button colours in HighContrastMode

### ISSUE THREAD

**Issue by ibrosen** [microsoft/fluentui#14683]
Link without an href takes button colours in HighContrastMode

### Environment Information

- **Package version(s)**: @fluentui/react ^7.117.1, @uifabric/styling ^7.12.15
- **Browser and OS versions**: Version 81.0.410.1 (Official build) Dev (64-bit) (repros on other browsers)

### Please provide a reproduction of the bug in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/link

#### Actual behavior:

![image](https://user-images.githubusercontent.com/24663638/90916867-bd5fe780-e396-11ea-8ea5-4038b1e24e04.png)

#### Expected behavior:

In my opinion, Link should have the same colour regardless of whether it has an Href (renders an anchor) or has no Href (renders a button)

### Priorities and help requested:

Are you willing to submit a PR to fix? **Yes**

Requested priority: **Normal**

Products/sites affected: **account.microsoft.com**

**Comment by Sandhya-aluri** [comment]
hi @msft-github-bot , we are using fluent/react. i have changed the version as you mentioned to "@fluentui/react": "^7.132.0" , but the change is not applying. still the color of the link without href in high contrast modes is same as not differentiating. can you please provide the fluent/react version.
